pkg/ebpf/telemetry/errors_telemetry_test.go

```diff
@@ -62,16 +78,31 @@ func skipTestIfEBPFTelemetryNotSupported(t *testing.T) {
 	}
 }
 
+func startManager(t *testing.T, m *manager.Manager, options manager.Options, name string, buf bytecode.AssetReader) {
```

Comment:
why don't we test the pkg/ebpf/manager wrapper here? (as it is actually used by the prod code)

Reply:
We cannot do that because it would cause an import cycle since `pkg/ebpf` imports `pkg/ebpf/telemetry`